editor/editor_help.cpp

```diff
@@ -56,19 +56,6 @@ void EditorHelp::_init_colors() {
 	class_desc->add_theme_constant_override("line_separation", Math::round(5 * EDSCALE));
 }
 
-void EditorHelp::_unhandled_key_input(const Ref<InputEvent> &p_ev) {
```

Comment:
FYI this was removed because the search popup is actually handled through `ScriptEditor::_menu_option`, via the `Search` menu button in the script editor. This code would never be hit.

![image](https://user-images.githubusercontent.com/41730826/93347699-b7b3cf80-f878-11ea-8d85-b83cd4d6626f.png)